framework/src/executioners/Transient.C

```diff
@@ -440,6 +447,17 @@ Transient::solveStep(Real input_dt)
 
       _solution_change_norm = _problem.solutionChangeNorm();
 
+      // Check for stop condition based upon steady-state check flag:
+      if (!_xfem_repeat_step &&
+          _trans_ss_check == true && _time > _ss_tmin)
+      {
+        // Compute new time solution l2_norm
+        Real new_time_solution_norm = _problem.getNonlinearSystem().currentSolution()->l2_norm();
+        // Compute l2_norm relative error
+        _ss_relerr_norm = fabs(new_time_solution_norm - _old_time_solution_norm)/new_time_solution_norm;
```

Comment:
Actually... this isn't the right thing to do!  I can see that it's the same formula as before... but it's just not right (and this code might be from me).

The formula SHOULD be:

norm(new_u - old_u) / norm(new_u)

What we currently have is:

(norm(new_u) - norm(old_u)) / norm(new_u)

Which is just plain wrong!